### ISSUE TITLE
Buttons are not working... 

### ISSUE THREAD

**Issue by huzaifaazim0** [skrafft/react-native-jitsi-meet#331]
i am on 
```
"react-native": "0.66.4",
"react-native-jitsi-meet": "^2.1.2"
```

and not a single button is working
which versions should i use? so that buttons work properly.

i used the "exampleApp" buttons were working fine but disabling chat feature didnt work
 
 i need this working with chat features disabled...

**Comment by andika-andriana** [comment]
hi @huzaifaazim0 ,

I thing your issue relate with [this](https://github.com/jitsi/jitsi-meet/issues/8948#issue-850157509) 
I facing same problem and solve that by downgrading my react-native version to 0.63

**Comment by vphutane27** [comment]
I am facing the same problem, not a single button working

**Comment by huzaifaazim0** [comment]
> hi @huzaifaazim0 ,
> 
> I thing your issue relate with [this](https://github.com/jitsi/jitsi-meet/issues/8948#issue-850157509) I facing same problem and solve that by downgrading my react-native version to 0.63

yes 0.63 works but while using 0.63 i get about 25 high level vulnerabilities...
if someone can give me a hint what breaks this lib in 0.64 then maybe i can spare some time to add support for 0.64

**Comment by rizbud** [comment]
same here. downgrade react-native version is not the best solution

**Comment by dgreasi** [comment]
Got it working with the following:

```
"react-native": "0.64.2",
```
but it was neccesery to build the `jitsi-meet` library myself, and update the `react-native` that `jitsi-meet` uses to `0.64.2`.

The error occurs because of [that](https://github.com/jitsi/jitsi-meet/issues/8948#issuecomment-856566676).

In short, the reason is that `3.6.0` sdk of `jitsi-meet` uses a `react-native` version < `0.64.0`, in which `react-native` uses the `playTouchSound` function of `Touchable`, which is no longer imported from the same library in versions > `0.64.0`

Thus no button in Android works because of the error.

**Comment by paudeldileep** [comment]
> Got it working with the following:
> 
> ```
> "react-native": "0.64.2",
> ```
> 
> but it was neccesery to build the `jitsi-meet` library myself, and update the `react-native` that `jitsi-meet` uses to `0.64.2`.
> 
> The error occurs because of [that](https://github.com/jitsi/jitsi-meet/issues/8948#issuecomment-856566676).
> 
> In short, the reason is that `3.6.0` sdk of `jitsi-meet` uses a `react-native` version < `0.64.0`, in which `react-native` uses the `playTouchSound` function of `Touchable`, which is no longer imported from the same library in versions > `0.64.0`
> 
> Thus no button in Android works because of the error.

can you explain detailed solution. What should i do inorder to make it work?

**Comment by dgreasi** [comment]
First you need a lib like this to consume the artifacts in a react-native application.

If you want to build the lib yourself (which is the case), to consume your artifacts you will need to override the sources of the react-native-jitsi-meet.

This can be done withing you project.

1) Android at your android/build.gradle
```
allprojects {
  repositories {
    ...
    
   maven {
       url "https://github.com/{your-repos}/raw/main/releases"
   }

    ...
  }
}

``` 

2) iOS in the end of your Podfile
```
...
pod 'JitsiMeetSDK', :git => 'https://github.com/{your-repos}.git', :tag => '{your-tag}'
```

Thus with the above lines you can use the builds (artifacts) that you created and override what the lib uses.

Note that in Android you need to specify folder releases from a branch and in iOS you can create a release in the repo and use the tag.

You can find more info [here](https://github.com/moirognwmonio/jitsi-meet/tree/sdk-3.6.0-mod) about the jitsi-meet build and the file structure of the repositories that contain the artifacts for each platform.

[This](https://github.com/moirognwmonio/jitsi-meet/tree/sdk-3.6.0-mod) is the jitsi-meet i used to build my artifacts, after the changes i made.

I also use my own [react-native-jitsi-meet](https://github.com/moirognwmonio/react-native-jitsi-meet) to use the jitsi-meet sdk-3.6.0, which is the one that worked for my project. Obviously, you can fork everything again and try to use a bigger version if you like.

I made my changes in the jitsi-meet project from the tag [ios-sdk-3.6.0](https://github.com/jitsi/jitsi-meet/releases/tag/ios-sdk-3.6.0).

Be aware that this process consumes a lot of time. Good luck.

**Comment by paudeldileep** [comment]
> First you need a lib like this to consume the artifacts in a react-native application.
> 
> If you want to build the lib yourself (which is the case), to consume your artifacts you will need to override the sources of the react-native-jitsi-meet.
> 
> This can be done withing you project.
> 
> 1. Android at your android/build.gradle
> 
> ```
> allprojects {
>   repositories {
>     ...
>     
>    maven {
>        url "https://github.com/{your-repos}/raw/main/releases"
>    }
> 
>     ...
>   }
> }
> ```
> 
> 2. iOS in the end of your Podfile
> 
> ```
> ...
> pod 'JitsiMeetSDK', :git => 'https://github.com/{your-repos}.git', :tag => '{your-tag}'
> ```
> 
> Thus with the above lines you can use the builds (artifacts) that you created and override what the lib uses.
> 
> Note that in Android you need to specify folder releases from a branch and in iOS you can create a release in the repo and use the tag.
> 
> You can find more info [here](https://github.com/moirognwmonio/jitsi-meet/tree/sdk-3.6.0-mod) about the jitsi-meet build and the file structure of the repositories that contain the artifacts for each platform.
> 
> [This](https://github.com/moirognwmonio/jitsi-meet/tree/sdk-3.6.0-mod) is the jitsi-meet i used to build my artifacts, after the changes i made.
> 
> I also use my own [react-native-jitsi-meet](https://github.com/moirognwmonio/react-native-jitsi-meet) to use the jitsi-meet sdk-3.6.0, which is the one that worked for my project. Obviously, you can fork everything again and try to use a bigger version if you like.
> 
> I made my changes in the jitsi-meet project from the tag [ios-sdk-3.6.0](https://github.com/jitsi/jitsi-meet/releases/tag/ios-sdk-3.6.0).
> 
> Be aware that this process consumes a lot of time. Good luck.

thanks for the info

**Comment by paudeldileep** [comment]
how do i join a call started by someone else?

**Comment by rcidt** [comment]
Try this solution https://github.com/skrafft/react-native-jitsi-meet/issues/265#issuecomment-1065550409
